src/functionalTest/resources/scenarios/RIA-4932-aip-appeal-outcome-allowed.json

```diff
@@ -0,0 +1,96 @@
+{
+  "description": "RIA-4932 - AIP - appeal outcome - allowed",
+  "enabled": "{$featureFlag.homeOfficeGovNotifyEnabled}",
+  "request": {
+    "uri": "/asylum/ccdAboutToSubmit",
+    "credentials": "CaseOfficer",
+    "input": {
+      "id": 1111,
+      "eventId": "sendDecisionAndReasons",
+      "state": "decided",
+      "caseData": {
+        "template": "minimal-appeal-submitted.json",
+        "replacements": {
+          "journeyType": "aip",
+          "appellantInUk": "Yes",
+          "subscriptions": [
+            {
+              "id": "1",
+              "value": {
+                "subscriber": "appellant",
+                "email": "{$TEST_CITIZEN_USERNAME}",
+                "wantsEmail": "Yes",
+                "mobileNumber": "{$TEST_CITIZEN_MOBILE}",
+                "wantsSms": "Yes"
+              }
+            }
+          ],
+          "ariaListingReference": "LP/12345/2019",
+          "isDecisionAllowed": "dismissed",
```

Comment:
@burale this `dismissed` decision doesn't tie in with the test description on line 2.